api/types/accesslist/accesslist.go

```diff
@@ -235,17 +236,17 @@ func (a *AccessList) MatchSearch(values []string) bool {
 }
 
 func (a *Audit) UnmarshalJSON(data []byte) error {
-	var audit map[string]string
+	var audit map[string]interface{}
```

Comment:
It's already merged but why don't you just use a temp struct to unmarshal these values. The unmarshal to `map[string]any` is not good when we know the types and fields. 
Extending to other fields isn't trivial and the current form is quite difficult to understand

Reply:
I'm happy to look into that as a follow on PR.

Reply:
Agree with this ^